ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -160,13 +160,16 @@ fi
 
 cat > ${cluster_config_file} << EOF
 {
+  "uuid": "$(uuidgen)",
```

Comment:
this is the uuid of the json file generated, it is not related to the cluster uuid. an uuid on the document is required to be uploaded to ES

Reply:
Thanks. How about make the item's name more clearly, e.g. es_uuid?

Reply:
I think uuid is better, ES is only the backend, if we want to use another backend, es_uuid makes no sense

Reply:
You are trying to add a non-existed item of the cluster detail in the cluster-config file, the cluster-config file is also used by the other team. We need to have an obvious naming for the new attribute. Otherwise, as you said, the uuid is a generated randomly, please use the cluster's uuid to upload to ES.

Reply:
there is no cluster uuid when this block is executed, this is a unique uuid to identify the execution, it is not related to the cluster. if workload fails, and cluster is not installed, we still need a uuid to identify the execution, there is no reason to link execution uuid to cluster uuid

Reply:
removed uuid